check-ntservice/README.md

```diff
@@ -42,8 +42,9 @@ command = ["check-ntservice", "--service-name", "SERVICE_NAME"]
 ### Options
 
 ```
-  -s, --service-name= service name
-  -l, --list-service  list service
+  -s, --service-name=    service name
+  -E, --exclude-service= service name to exclude from matching
```

Comment:
I think traditional commands such as *diff*, *tar*, or *du* use often `x|X` character for meaning *exclude* instead of `e|E`, or no short notation.

Reply:
I was not certain the character, either. I follow your point, ee53b4b .